src/Jenssegers/Mongodb/Eloquent/Model.php

```diff
@@ -233,11 +234,11 @@ public function getAttribute($key)
 
                 // This attribute matches an embedsOne or embedsMany relation so we need
                 // to return the relation results instead of the interal attributes.
-                if ($relations instanceof EmbedsOneOrMany) {
+                if ($relations instanceof EmbedsOneOrMany or $relations instanceof MorphTo) {
```

Comment:
RelationsTest.php:367 : `Failed asserting that null is an instance of class "User".`
As a workaround to working MorphTo Relations will be directly returned and resolved.
Is it necessary to separate this logical part?